components/BottomNavigation/src/private/MDCBottomNavigationItemView.h

```diff
@@ -28,6 +28,13 @@
 @property(nonatomic, copy) NSString *badgeValue;
 @property(nonatomic, copy) NSString *title;
 @property(nonatomic, strong) UIFont *itemTitleFont UI_APPEARANCE_SELECTOR;
+
+/**
+ The number of lines available for rendering the title of this item.
+
+ @note This property is only used when @c titleBelowIcon is @c true.
```

Comment:
can you clarify that the default is 1 here?

Reply:
Done.